src/Bicep.LangServer.UnitTests/Configuration/BicepConfigChangeHandlerTests.cs

```diff
@@ -1,244 +0,0 @@
-// Copyright (c) Microsoft Corporation.
-// Licensed under the MIT License.
-
-using System;
-using System.Diagnostics.CodeAnalysis;
-using System.IO;
-using System.Linq;
-using Bicep.Core;
-using Bicep.Core.Configuration;
-using Bicep.Core.FileSystem;
-using Bicep.Core.UnitTests.Utils;
-using Bicep.Core.Workspaces;
-using Bicep.LanguageServer;
-using Bicep.LanguageServer.Configuration;
-using FluentAssertions;
-using Microsoft.VisualStudio.TestTools.UnitTesting;
-using Moq;
-using OmniSharp.Extensions.LanguageServer.Protocol;
-using OmniSharp.Extensions.LanguageServer.Protocol.Models;
-using OmniSharp.Extensions.LanguageServer.Protocol.Server;
-using Range = OmniSharp.Extensions.LanguageServer.Protocol.Models.Range;
-using IOFileSystem = System.IO.Abstractions.FileSystem;
-using Bicep.Core.Analyzers.Linter;
-using Bicep.Core.UnitTests;
-
-namespace Bicep.LangServer.UnitTests.Configuration
-{
-    [TestClass]
-    public class BicepConfigChangeHandlerTests
```

Comment:
What happened to these tests?

Reply:
This test class was only exercising the `RefreshCompilationOfSourceFilesInWorkspace` method. Since that behavior was moved to `BicepDidChangeWatchedFilesHandler`, the tests were moved to `BicepDidChangeWatchedFilesHandlerTests`